scanner/decisions/0005-scannerv4-read-cvss-data.md

```diff
@@ -0,0 +1,27 @@
+# 0005 - Adapt Scanner V4 NVD CVSS data updating pipeline to integrate with NVD CVE Api
+
+- **Author(s):** Yi Li <yli3@redhat.com>
+- **Created:** [2023-10-10]
+
+## Status
+
+Accepted
+
+## Context
+
+The National Vulnerability Database (NVD) has announced the retirement of its JSON feeds by December 15, 2023. This creates the need to change our existing CVSS data updater pipeline for Scanner V4, which currently relies on these JSON feeds.
+
+## Decision
+
+The current NVD CVSS GitHub Workflow will be updated to fetch data from the NVD CVE API instead of NVD Json feeds while maintaining the freshness of the NVD CVSS data bundle in Google Storage. It will categorize the downloaded data by CVSS V3 severity levels, and these categorized data will be archived into individual zip files, such as `severity-low.zip`. The format of the JSON is the same as before.
```

Comment:
> The current NVD CVSS GitHub Workflow will be updated to fetch data from the NVD CVE API instead of NVD Json feeds while maintaining the freshness of the NVD CVSS data bundle in Google Storage.

Please briefly explain the diff between the JSON endpoints and the NVD API. With that information, I could give you feedback on the decision to create individual ZIPs according to CVSS V3 severity levels. I am curious to understand why grouping by severity. And also, if there is any implication on the reliability of the workflows to handle API issues that might block the GitHub workflow from updating the bucket (e.g., rate limiting).